core/src/main/kotlin/com/tschuchort/compiletesting/AbstractKotlinCompilation.kt

```diff
@@ -262,7 +262,7 @@ abstract class AbstractKotlinCompilation<A : CommonCompilerArguments> internal c
             .mapNotNull { url ->
                 val uri = URI.create(url.toString().removeSuffix("/$resourceName"))
```

Comment:
I wonder why the same problem doesn't occur here already.

Reply:
I also had a same doubt